support-frontend/assets/pages/supporter-plus-landing/setup/legacyActionCreators.ts

```diff
@@ -112,12 +112,13 @@ const buildStripeChargeDataFromAuthorisation = (
 		state.common.abParticipations,
 		state.page.checkoutForm.billingAddress.fields.postCode,
 	),
-	publicKey: getStripeKey(
-		stripeAccountForContributionType[getContributionType(state)],
-		state.common.internationalisation.countryId,
-		state.common.internationalisation.currencyId,
-		state.page.user.isTestUser,
-	),
+	publicKey:
+		/* why don't we use state.page.checkoutForm.payment.stripeAccountDetails.publicKey ?*/ getStripeKey(
```

Comment:
this is the mechanism that sends the key over to payment api.  I wasn't quite sure why we call getStripeKey again here (with hopefully the same parameters), rather than just using the value from the state?